### ISSUE TITLE
State of Terraform provider

### ISSUE THREAD

**Issue by TobiPeterG** [nautobot/terraform-provider-nautobot#16]
Hey there, the company I work for recently switched from netbox to nautobot. We use opentofu/Terraform to provision our VMs and currently use the netbox provider to get free ip addresses, create the VM in netbox etc.

Is there any plan to support Terraform/opentofu in the future?

**Comment by chadell** [comment]
yes, there is a plan but the current provider is a draft version (i.e., only the manufacture endpoint was implemented plus the graphql query), and we are more than happy to get community contributions to it, the same was as NTC will contribute to it when we have opportunities.
@TobiPeterG , you could define the features requested as GitHub issues

**Comment by TobiPeterG** [comment]
> yes, there is a plan but the current provider is a draft version (i.e., only the manufacture endpoint was implemented plus the graphql query), and we are more than happy to get community contributions to it, the same was as NTC will contribute to it when we have opportunities.
> @TobiPeterG , you could define the features requested as GitHub issues


Hey, thanks for your reply. :)
When I saw it correctly, the Terraform provider uses go-nautobot, right?
And that doesn't support nautobot v2.x right?
I forked the repo and at least managed to fix the test environment and Link it against nautobot 2.1.9, feel free to have a look :)

The tests currently still fail though and unfortunately I don't have experience with oapi (yet)
